extensions/emmet/src/matchTag.ts

```diff
@@ -4,8 +4,11 @@
  *--------------------------------------------------------------------------------------------*/
 
 import * as vscode from 'vscode';
+import parse from '@emmetio/html-matcher';
```

Comment:
Since we end up using `parse` directly, let's replace the use of `parseDocument` in this file with `parse` to keep it consistent